smart-client-jersey/src/main/java/com/emc/rest/smart/jersey/SmartClientFactory.java

```diff
@@ -150,6 +164,12 @@ static ApacheHttpClient4Handler createApacheClientHandler(SmartConfig smartConfi
         connectionManager.setMaxTotal(smartConfig.getIntProperty(MAX_CONNECTIONS, MAX_CONNECTIONS_DEFAULT));
         clientConfig.getProperties().put(ApacheHttpClient4Config.PROPERTY_CONNECTION_MANAGER, connectionManager);
 
+        ScheduledExecutorService sched = Executors.newSingleThreadScheduledExecutor();
```

Comment:
The scheduler should only be created if `SmartConfig.maxConnectionIdleTime` is > 0